cluster/plan.go

```diff
@@ -363,34 +423,46 @@ func (c *Cluster) BuildEtcdProcess(host *hosts.Host, etcdHosts []*hosts.Host) v3
 	if host.ExistingEtcdCluster {
 		clusterState = "existing"
 	}
-	args := []string{"/usr/local/bin/etcd",
-		"--name=etcd-" + host.HostnameOverride,
-		"--data-dir=/var/lib/rancher/etcd",
-		"--advertise-client-urls=https://" + host.InternalAddress + ":2379,https://" + host.InternalAddress + ":4001",
-		"--listen-client-urls=https://0.0.0.0:2379",
-		"--initial-advertise-peer-urls=https://" + host.InternalAddress + ":2380",
-		"--listen-peer-urls=https://0.0.0.0:2380",
-		"--initial-cluster-token=etcd-cluster-1",
-		"--initial-cluster=" + initCluster,
-		"--initial-cluster-state=" + clusterState,
+	args := []string{
+		"/usr/local/bin/etcd",
 		"--peer-client-cert-auth",
 		"--client-cert-auth",
-		"--trusted-ca-file=" + pki.GetCertPath(pki.CACertName),
-		"--peer-trusted-ca-file=" + pki.GetCertPath(pki.CACertName),
-		"--cert-file=" + pki.GetCertPath(nodeName),
-		"--key-file=" + pki.GetKeyPath(nodeName),
-		"--peer-cert-file=" + pki.GetCertPath(nodeName),
-		"--peer-key-file=" + pki.GetKeyPath(nodeName),
+	}
+
+	CommandArgs := map[string]string{
+		"--name":                        "etcd-" + host.HostnameOverride,
```

Comment:
the etcd command arguments here are wrong, because you already add `--` to each line, after testing it i get the following:
```
bad flag syntax: ----name=****
```